packages/components/src/Inventory/AppInfo.js

```diff
@@ -1,26 +1,25 @@
 import React, { Suspense } from 'react';
 import PropTypes from 'prop-types';
 import { ScalprumComponent } from '@scalprum/react-core';
-import { useHistory } from 'react-router-dom';
 import { useStore } from 'react-redux';
 import { Bullseye, Spinner } from '@patternfly/react-core';
 import InventoryLoadError from './InventoryLoadError';
 import classNames from 'classnames';
+import WithHistory from './WithHistory';
 
 const BaseAppInfo = (props) => {
```

Comment:
Maybe at some point. But for now I think it is better using them, because otherwise applications would need to import scalprum itself and we'd probably have to make sure to keep those versions aligned.